### ISSUE TITLE
Accessibility: Combobox: Group headers within the combo box list are not reachable via keyboard. 

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3456]
Description: Group headers within the combo box list are not reachable via keyboard. 

Expected: Group headers within the combo box list should be reachable via keyboard

Screenshot:

![image](https://user-images.githubusercontent.com/32538291/94523088-5283b500-024e-11eb-9c6a-f3a1fe239d44.png)

![image](https://user-images.githubusercontent.com/32538291/94523101-557ea580-024e-11eb-8795-d80cf32f9053.png)

**Comment by Sanjay-1312** [comment]
Hello Colleagues,

The issue is not fixed.
The group header name is not read.
So please fix the issue.
URL: https://fundamental-ngx.netlify.app/#/platform/combobox
JAWS 2021

Screenshot:
![image](https://user-images.githubusercontent.com/78714518/119931939-fcfbd580-bf9f-11eb-8bd1-31d43f269093.png)

**Comment by Betrozov** [comment]
Hi @Sanjay-1312
please test using this link
https://deploy-preview-5206--fundamental-ngx.netlify.app/

**Comment by Sanjay-1312** [comment]
The Issue still persists i can able to reproduce the issue.
The list header such as 'Vegetable' and 'Fruits' is not read
when accessed it is read as 'Open List box' when 'Vegetable is accessed and nothing is read on accessing the 'Vegetable' Header.

![image](https://user-images.githubusercontent.com/78714518/120318689-c3a7ca80-c2fd-11eb-9faa-97f4d24b6e7b.png)

URL:https://deploy-preview-5206--fundamental-ngx.netlify.app/
JAWS 2021

**Comment by puru-hk** [comment]
@Betrozov Please see the above comments from the accessibility team.

**Comment by olshansky** [comment]
Assigned to @OleksiyDrozdov

**Comment by Sanjay-1312** [comment]
Issue:
OB: There is no visible focus provided for the 'Fruits' and 'vegetables'.
EB: Ideally, four sided focus should be provided so that user will be able to identify on which element the focus is available.
![image](https://user-images.githubusercontent.com/78714518/127625695-80e44166-9f10-4f02-9283-d91ba71f1b50.png)
![image](https://user-images.githubusercontent.com/78714518/127625778-f74de421-cdad-4c0c-ba5a-5070616503b8.png)
